apps/client/src/hoc/withTracking.js

```diff
@@ -18,11 +18,11 @@ const withTracking = (Component) => ({ ...props }) => {
     }
   };
 
-  const matomoTrackEvent = ({ action, category = topic.name, name }) => {
+  const matomoTrackEvent = ({ action, category, name }) => {
     if (trackingEnabled()) {
       trackEvent({
         action: action.toLowerCase(),
-        category: category.toLowerCase(),
+        category: (category || topic?.name || slug || "unknown").toLowerCase(),
```

Comment:
I don't think we should track "unknown" category sources. Maybe only on acceptance, but not on production. This makes us need a filter `where category !== 'unknown'`

Reply:
I've added this to the frontend board. We can pick this up.